src/cargo/ops/vendor.rs

```diff
@@ -618,10 +618,6 @@ fn vendor_this(relative: &Path) -> bool {
         // Temporary Cargo files
         Some(".cargo-ok") => false,
 
-        // Skip patch-style orig/rej files. Published crates on crates.io
-        // have `Cargo.toml.orig` which we don't want to use here and
-        // otherwise these are rarely used as part of the build process.
-        Some(p) if p.ends_with(".orig") || p.ends_with(".rej") => false,
```

Comment:
What are `.rej` files?

Reply:
From GNU patch

> The names for reject files (files containing patches that patch could not find a place to apply) are normally the name of the output file with ‘.rej’ appended (or ‘#’ if using ‘.rej’ would make the backup file name too long). 

https://www.gnu.org/software/diffutils/manual/html_node/Reject-Names.html

Reply:
I feel like if the file is inside the `.crate`, we might as well expose it, so this seems fine.